assets/i18n/en.json

```diff
@@ -668,41 +678,57 @@
     "I18N_LEARNER_GROUP_CREATION_NEXT_BUTTON_TEXT": "Next",
     "I18N_LEARNER_GROUP_CREATION_PREV_BUTTON_TEXT": "Previous Step",
     "I18N_LEARNER_GROUP_DETAILS_GROUP_DESCRIPTION": "Group Description (Describe the group objectives in 2-4 lines)",
     "I18N_LEARNER_GROUP_DETAILS_GROUP_TITLE": "Group Title",
+    "I18N_LEARNER_GROUP_DETAILS_MODAL_DESCRIPTION": "Group Description",
+    "I18N_LEARNER_GROUP_DETAILS_TITLE": "Details",
     "I18N_LEARNER_GROUP_GROUP_DETAILS_SECTION": "Group Details",
+    "I18N_LEARNER_GROUP_INVITATION_MODAL_HEADER": "Learner group invitation",
     "I18N_LEARNER_GROUP_INVITE_LEARNERS": "Invite Learners",
     "I18N_LEARNER_GROUP_INVITE_LEARNERS_BY_USERNAME": "Invite learners through username",
     "I18N_LEARNER_GROUP_INVITE_LEARNERS_PLACEHOLDER_TEXT": "Add username of the learner to invite and press enter",
     "I18N_LEARNER_GROUP_INVITE_LIST_TEXT": "Invite List",
     "I18N_LEARNER_GROUP_ITEM_ADDED_TO_SYLLABUS": "Added",
     "I18N_LEARNER_GROUP_ITEM_ALREADY_ADDED_TO_SYLLABUS": "Already In Syllabus",
+    "I18N_LEARNER_GROUP_JOINING_MESSAGE": "You are about to join \"<[groupName]>\"",
     "I18N_LEARNER_GROUP_LEARNERS": "{learnersCount, plural, =1{LEARNER} other{LEARNERS}}",
+    "I18N_LEARNER_GROUP_LEARNERS_MODAL_TEXT": "Learners",
     "I18N_LEARNER_GROUP_LEARNERS_PROGRESS_TAB": "Learners' Progress",
     "I18N_LEARNER_GROUP_LEARNERS_SECTION": "{learnersCount, plural, =1{Learner} other{Learners}}",
     "I18N_LEARNER_GROUP_MINIMUM_SYLLABUS_ITEMS_INFO": "You need to add at least one syllabus item(Skill/Story) in order to make a group.",
+    "I18N_LEARNER_GROUP_NO_INVITATIONS": "You do not have any pending invitations.",
     "I18N_LEARNER_GROUP_NO_ITEMS_ADDED": "You haven't added new items yet.",
     "I18N_LEARNER_GROUP_NO_LEARNERS_HAVE_JOINED": "No learners have joined the group yet.",
     "I18N_LEARNER_GROUP_NO_LEARNERS_INVITED": "You haven't invited any learners yet.",
     "I18N_LEARNER_GROUP_NO_RESULTS_FOUND": "No results found.",
     "I18N_LEARNER_GROUP_OVERVIEW_TAB": "Overview",
+    "I18N_LEARNER_GROUP_PAGE_TITLE": "Learner Group | Oppia",
     "I18N_LEARNER_GROUP_PERMISSION_NOT_GIVEN": "Permission Not Given",
     "I18N_LEARNER_GROUP_PREFERENCE": "Learner Group Preference",
+    "I18N_LEARNER_GROUP_PREFERENCES_MODAL_SAVE_BUTTON": "Save",
     "I18N_LEARNER_GROUP_PREFERENCES_TAB": "Preferences",
     "I18N_LEARNER_GROUP_PROGRESS_IN_STORIES_SECTION": "Progress in Stories",
     "I18N_LEARNER_GROUP_PROGRESS_NO_LEARNERS": "There are no learners in this group. Please invite learners to see their progress.",
+    "I18N_LEARNER_GROUP_PROGRESS_SHARING_INFO_TEXT": "You can allow the facilitators to view your progress in assigned skills and/or stories. By giving the group owner these permissions, the group owner will be able to gain insights into your progress and better support your learning. You can change this setting anytime.",
```

Comment:
Is "facilitators" the same thing as "group owners"?

Reply:
yes, they are the same.